src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -5447,10 +5505,13 @@ void *wm_vuldet_main(wm_vuldet_t* vuldet) {
                     if (OS_INVALID == register_task(task)) {
                         mwarn("Couldn't establish communication with task manager. There's no ID for task");
                     }
+                    mdebug2("Scheduling task type '%s' ID: '%d'", vu_task_type_tag[task->task_type], task->task_id);
+                    vuldet->last_scan = time(NULL);
                     linked_queue_push_ex(vu_tasks_queue, task);
+                } else {
+                    mwarn("Scan by interval already scheduled in tasks queue");
+                    vuldet->last_scan = time(NULL);
```

Comment:
If we write this outside of both the **if** and the **else**, we can write it only one time.

Reply:
Of course, thank you!